packages/blockchain-wallet-v4-frontend/src/components/Form/AmountFieldInput/index.tsx

```diff
@@ -63,6 +64,8 @@ const AmountFieldInput: React.FC<Props> = ({
     return formatTextAmount(value, allValues && allValues.fix === 'FIAT')
   }
 
+  const { coinfig } = window.coins[coin || 'BTC']
```

Comment:
Can we use `getCoins` selector here instead? We shall avoid using `window.coins` directly going forward.